medusa/config.py

```diff
@@ -440,6 +440,32 @@ def change_PROCESS_AUTOMATICALLY(process_automatically):
         app.auto_post_processor_scheduler.silent = True
 
 
+def change_REMOVE_FROM_CLIENT(remove_from_client):
```

Comment:
Please keep `change_REMOVE_FROM_CLIENT` all lowercase. I know the other functions are also like this, but it's not like it should be.

I'd also rename the parameter `remove_from_client` to `new_state` or something more explicit.